### PR TITLE
Aswathy/DPROD-2695/Converting the font from static to google static font

### PR DESCRIPTION
**Pull request by aswathy-deriv**
Changes:

-   Optimize this font files and package all font files into a single one. 

## Type of change

-   [ ] Bug fix
-   [x] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: static/css/ubuntu.css [hunk 1]
```diff
@@ -1,27 +1,28 @@
...
     font-weight: 200;
 }
 
 @font-face {
     font-family: 'Ubuntu';
-    src:url('../fonts/Ubuntu-Medium.woff2') format('woff2');
+    src: url('../fonts/Ubuntu-Medium.woff2') format('woff2');
     font-display: swap;
     font-weight: 500;
 }
```
Comment (dan-deriv) [inline static/css/ubuntu.css]: Why do we have some font-faces not from google?

Comment (aswathy-deriv) [reply static/css/ubuntu.css]: I think you are viewing the outdated file.Can you check the new commit for the file

Comment (dan-deriv) [reply static/css/ubuntu.css]: Perfect, thanks! Just walked through commits )

File: static/css/ubuntu.css [hunk 2]
```diff
@@ -1,27 +1 @@
...
-    font-weight: 200;
-}
-
-@font-face {
-    font-family: 'Ubuntu';
-    src:url('../fonts/Ubuntu-Medium.woff2') format('woff2');
-    font-display: swap;
-    font-weight: 500;
-}
+@import url('https://fonts.googleapis.com/css2?family=Ubuntu:wght@400&display=swap');
```
Comment (habib-deriv) [inline static/css/ubuntu.css:1]: Ubunto 700 not needed?
```suggestion
@import url('https://fonts.googleapis.com/css2?family=Ubuntu:wght@400&display=swap');
```

Comment (aswathy-deriv) [reply static/css/ubuntu.css:1]: Ubuntu 700 is coming from the quill-tailwind font css @habib-deriv

**Review by habib-deriv — APPROVED** [review]

**Review by meenakshi-deriv — APPROVED** [review]

**Review by mitra-deriv — APPROVED** [review]

**Pull request merged**